Show installing Foreman Server for all builds (#4188)
theforeman/foreman-documentation: guides/common/modules/proc_restoring-manual-changes-overwritten-by-a-puppet-run.adoc

The following example shows you how to restore a DHCP configuration file overwritten by a Puppet run.

.Procedure
. Create a backup of the configuration file that you want to restore:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# cp /etc/dhcp/dhcpd.conf /etc/dhcp/dhcpd.backup
----
+

> This allows you to compare the files to check for any mandatory changes required by the upgrade.

Reviewer: ```suggestion
You can compare the overwritten file with the backup to check for any mandatory changes required by the upgrade.
```

Author: Applied.
